samples/databases/wide-world-importers/wwi-ssdt/README.md

```diff
@@ -0,0 +1,98 @@
+
+### Contents
+
+[About this sample](#about-this-sample)<br/>
+[Before you begin](#before-you-begin)<br/>
+[Run this sample](#run-this-sample)<br/>
+[Disclaimers](#disclaimers)<br/>
+
+
+<a name=about-this-sample></a>
+
+## About this sample
+
+<!-- Delete the ones that don't apply -->
+1. **Applies to:** SQL Server 2016 (or higher), Azure SQL Database [testing and modified instructions are TBD]
+1. **Key features:** Core database features
+1. **Workload:** OLTP
+1. **Programming Language:** Transact-SQL
+1. **Authors:** Greg Low, Denzil Ribeiro, Jos de Bruijn, Robert Cain
+
+The instructions below are for creating the sample database from scratch.
+
+A pre-created version of the database is available for download as part of the latest release of the sample: [wide-world-importers-release](http://go.microsoft.com/fwlink/?LinkID=800630).
+
+<a name=before-you-begin></a>
+
+## Before you begin
+
+To run this sample, you need the following prerequisites.
+
+**Software prerequisites:**
+
+1. SQL Server 2016 SP1 (or higher) or an Azure SQL Database. Also works with SQL Server 2016 RTM, for Evaluation, Developer, and Enterprise edition.
+2. Visual Studio 2015 Update (or higher) with SQL Server Data Tools (SSDT). We recommend you update to the latest available of SSDT from the Visual Studio Extensions and Updates feed.
+
+
+<a name=run-this-sample></a>
+
+## Run this sample
+
+The below steps reconstruct the WideWorldImporters database.
+
+Note that each time the database is created from scratch, the data in many tables will be different as a degree of randomization is used throughout the code.
+
+<!-- Step by step instructions. Here's a few examples -->
+
+### Publishing to SQL Server
+
+1. Open the solution **wwi-ssdt.sln** in Visual Studio. Skip this step if you have already opened the solution **wwi-sample.sln** in the root of this sample.
+
+2. Build the solution.
+
+3. Publish the WideWorldImporters database:
+    1. In the Solution Explorer, right-click the **WideWorldImporters** project, and select **Publish** to bring up the **Publish Database** dialog.
+    1. Click **Edit** to modify the **Target Database Connection** to point to your SQL Server 2016 (or later) instance.
+    1. Edit the **Database Name** to say "WideWorldImporters".
+    1. Click **Publish**.
+    1. Wait for publication to finish. You can monitor progress in the **Data Tools Operations** page in Visual Studio. In testing this took around 3 minutes.
+
+4. (Optional) Data population: After step 3, the database contains data for January 2013. This step populates data from February 2013 up to the current data.
+    1. Open SQL Server Management Studio, and connect to the WideWorldImporters database that was published in the previous step.
+    1. Execute the following script. This may take a while to complete - populating data from Feb 2013 to Jun 2016 took about 40 minutes in one test.
```

Comment:
It might be worthwhile considering adding some machine specs for instance, "on a machine with 30GB RAM and 4 core the load took about 40 minutes (this is just for estimation purposes, the actual time taken may vary from machine to machine)"?